string prior_name
point_cloud_segmentation/GraspObject seg_obj
HandConfig init_hand_config
string grasp_type
---
float32[] inf_config_array
prob_grasp_planner/HandConfig full_inf_config
float32[] init_config_array
prob_grasp_planner/HandConfig full_init_config
float32 inf_val
float32 inf_suc_prob
float32 inf_log_prior
float32 init_val
float32 init_suc_prob
float32 init_log_prior
int32[] sparse_voxel_grid
float32 voxel_size
int32[] voxel_grid_dim
float32[] object_size
bool success

